hugolib/skeletons/site/archetypes/default.md

```diff
@@ -0,0 +1,5 @@
++++
+title = '{{ replace .File.ContentBaseName "-" " " | strings.Title }}'
+date = {{ .Date }}
+draft = true
++++
```

Comment:
This is not very important, but is there a good reason why this skeleton package lives below hugolib? I would think that `create/skeletons` would be better.